help/en/package/apps/TabbedPreferences.shtml

```diff
@@ -279,7 +279,7 @@ This will let you to limit what users have the ability to control.
 </dl>
 
 The "Apply" button puts the current settings into effect, without
-saving them permanently.  For a connected throttle to use the
+saving them permanently.  For a connected <a href="../jmri/jmrit/withrottle/UserInterface.shtml">WiThrottle</a> to use the
```

Comment:
You're right. If the Port Number pref is changed while WiThrottle Server is running, just restarting WiThrottle and the iPad app doesn't work as it chokes the WiThrottle Light app. So we have to advice people to either completely restart JMRI after changing the WiThrottle Port Number (when running) or only change the Port Number after stopping server. Then, it's a much easier process: Just release your WiThrottle Light engine (it would have been disconnected anyway as the server was down) and set it again on the mobile device.